pkg/standalone-utils/contracts/relayer/interfaces/IMockEulerProtocol.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+
+interface IMockEulerProtocol {
+    /**
+     * @dev triggers a transferFrom call with `from` being the msg.sender
+     * @notice This mimics the requirement to ensure the euler protocol
+     * is allowed to transfer from msg.sender
+     */
+    function requestUnderlyingFromRelayer(
+        address underlying,
```

Comment:
Can import `IERC20` in this file and make these arguments of type `IERC20` instead of `address` to avoid casting later.